### PR TITLE
reference: linux-targets: fix generic rv64 target image

### PR DESCRIPTION
**Pull request by mbolivar**
The .otaimg file is .ota-ext4 now.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 063dc2c21dd0a26d5827dd7c52a3cb55cf365ba4 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/275/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 68e7d8d41e295dd9bd2589d1b02c39c87f38cea3 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/276/docs/artifacts/html/index.html

File: source/reference/linux-targets.rst [hunk 1]
```diff
@@ -327,40 +327,30 @@ Generic RISC-V 64 Machine
...
 
-     brew install qemu
-
-   To install latest QEMU on Ubuntu 18.04, run::
-
-     sudo add-apt-repository ppa:fio-maintainers/riscv
-     sudo apt-get update
-     sudo apt-get install qemu-system-misc
+   The artifacts required by QEMU are ``bbl`` (Berkeley Boot Loader +
+   Kernel + initrd) and ``lmp-mini-image-qemuriscv64.ota-ext4`` in
```
Comment (ricardosalveti) [inline source/reference/linux-targets.rst]: 'Initrd' instead of 'initrd'?

File: source/reference/linux-targets.rst [hunk 2]
```diff
@@ -327,40 +327,30 @@ Generic RISC-V 64 Machine
...
+   ``deploy/images/qemuriscv64``..
```
Comment (ricardosalveti) [inline source/reference/linux-targets.rst]: Extra dot.

File: source/reference/linux-targets.rst (hunk 1 above)
Comment (mbolivar) [reply source/reference/linux-targets.rst]: The usual spelling seems to be initrd: https://en.wikipedia.org/wiki/Initial_ramdisk

I can put it back to how it was before

File: source/reference/linux-targets.rst (hunk 2 above)
Comment (mbolivar) [reply source/reference/linux-targets.rst]: Fixed, thanks

**Comment by doanac** [comment]
Docs for b22db660b648adb016f64c2f12cca55e5c064479 are browesable at: https://ci.foundries.io/projects/fio-docs/builds/277/docs/artifacts/html/index.html

File: source/reference/linux-targets.rst (hunk 1 above)
Comment (ricardosalveti) [reply source/reference/linux-targets.rst]: Nops, no need :-)

**Pull request merged**